packages/coinstac-ui/app/render/components/consortium/consortium-results.js

```diff
@@ -92,3 +110,11 @@ ConsortiumResults.compareRemoteResults = (
 
   return 0;
 };
+
+const mapStateToProps = ({ consortiaPage }) => {
```

Comment:
You can destructure this directly in the parameter:

```js
const mapStateToProps = ({
  consortiaPage: { expandedResult },
}) => {
  /* ... */
};
```

Avoids the variable declaration line.